doc/user/content/sql/show-index.md

```diff
@@ -26,13 +26,14 @@ _on&lowbar;name_ | The name of the object whose indexes you want to show. This c
 `SHOW INDEX`'s output is a table, with this structure:
 
 ```nofmt
- on_name | key_name | seq_in_index | column_name | expression | nullable | enabled
----------+----------+--------------+-------------+------------+----------+--------
- ...     | ...      | ...          | ...         | ...        | ...      | ...
+cluster | on_name | key_name | seq_in_index | column_name | expression | nullable | enabled
+--------+---------+----------+--------------+-------------+------------+----------+--------
+ ...    | ...     | ...      | ...          | ...         | ...        | ...      | ...
 ```
 
 Field | Meaning
 ------|--------
+**cluster** | The name of the [cluster](/overview/api-components/#clusters) containing the sink
```

Comment:
Iff you back through CI, this should terminate with a period.

Reply:
Ha! I agree! I was trying to follow the style in the `SHOW SINKS` table, but looks like that whole table was actually messing this up. I'll fix.

Reply:
Fixed now.